benefits/static/css/admin/styles.css

```diff
@@ -90,9 +102,89 @@ html[data-theme="light"],
   row-gap: 1rem;
 }
 
-/* Enrollment */
+/* Enrollment Page*/
 iframe.card-collection {
   border: 0px;
   width: 100%;
   height: 60vh;
 }
+
+/* Login Page */
+.login #header {
+  padding: 0 !important;
+}
+
+.login #branding {
+  flex-direction: column;
+  width: 100%;
+}
+
+.login #container {
+  background: var(--body-bg);
+  border: 1px solid var(--hairline-color);
+  overflow: hidden;
+  width: 28em;
+  min-width: 300px;
+  margin: 100px auto;
+  height: auto;
+}
+
+.login .form-row label {
+  display: block;
+  line-height: 2em;
+}
+
+.login .form-row #id_username,
+.login .form-row #id_password {
+  padding: 8px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.login .password-reset-link {
+  text-align: center;
+}
```

Comment:
This is directly from https://github.com/django/django/blob/9ca1f6eff6f19d1ae074d289c6c4209073351805/django/contrib/admin/static/admin/css/login.css#L59 but I don't know how to trigger it or if it shows up in our app

Reply:
If your database has the user from the sample fixtures (CST User) and you log in as that user, you'll see a "Change password" link in the usertools section.

![image](https://github.com/user-attachments/assets/08ac4b50-0579-4870-92d8-961edb8521b7)   


... speaking of which, that link takes us to a page that we have not overridden / that doesn't use our `extrastyles`, so it uses all the default Django styles. 

![image](https://github.com/user-attachments/assets/2c376c35-4904-4905-94ad-4262c99e60e8)

I'm not sure if what we should have this link or if we should override it to use our extra styles. I'll create a separate ticket for this.

Reply:
Looks like the link shows up based on if the user "has a changeable password":

https://github.com/django/django/blob/1857b6663b664847a426e8c79b8aec42994660ba/django/contrib/admin/templates/admin/base.html#L56-L58

Reply:
Created #2384

Reply:
Oh wait, `password-reset-link` is actually a different thing from what I was talking about.

It's a link on the login page. I guess we wouldn't show it because we don't have a `'admin_password_reset'` route defined.

https://github.com/search?q=repo%3Adjango%2Fdjango%20password-reset-link&type=code

https://github.com/django/django/blob/1857b6663b664847a426e8c79b8aec42994660ba/django/contrib/admin/templates/admin/login.html#L59-L61

Reply:
https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#adding-a-password-reset-feature

I don't think we should worry about adding this now since we ideally want to use SSO long-term

Reply:
On the other hand, any kind of self-service we can offer to users _while_ we have to manage them locally would be handy. 

Let's create another ticket to figure out what to do about the Reset link on the login page.

Reply:
Created https://github.com/cal-itp/benefits/issues/2386 for the reset password link feature